lib/extractors/exported.js

```diff
@@ -1,40 +1,93 @@
 var traverse = require('babel-traverse').default,
-  isJSDocComment = require('../../lib/is_jsdoc_comment');
-
+  isJSDocComment = require('../../lib/is_jsdoc_comment'),
+  t = require('babel-types'),
+  nodePath = require('path'),
+  fs = require('fs');
 
 /**
  * Iterate through the abstract syntax tree, finding ES6-style exports,
  * and inserting blank comments into documentation.js's processing stream.
  * Through inference steps, these comments gain more information and are automatically
  * documented as well as we can.
+ * @param {Function} parseToAst funtiont that parses to an ast
  * @param {Object} ast the babel-parsed syntax tree
+ * @param {Object} data the name of the file
  * @param {Function} addComment a method that creates a new comment if necessary
  * @returns {Array<Object>} comments
  * @private
  */
-function walkExported(ast, addComment) {
+function walkExported(parseToAst, ast, data, addComment) {
```

Comment:
It is questionable if passing in `addComment` is worth the complexity here. I'll look at that too.